calico-enterprise/getting-started/upgrading/upgrading-enterprise/kubernetes-upgrade-tsee/helm.mdx

```diff
@@ -144,11 +144,13 @@ These steps differ based on your cluster type. If you are unsure of your cluster
       ? (
         `helm upgrade calico-enterprise --reuse-values --values=<path-to-values-yaml> tigera-operator-v0.0.tgz \\
   --set-file imagePullSecrets.tigera-pull-secret=<path-to-pull-secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path-to-pull-secret> \\
+  --set-file licenseKeyContent=<path/to/license/file/yaml> \\
```

Comment:
```suggestion
  --set-file licenseKeyContent=<path-to-license-file-yaml> \\
```
Just to follow the convention already in this code block and elsewhere in the documentation. Otherwise LGTM.

Reply:
@ctauchen Thanks! The paths were updated to using the convention. The commits were squashed